server/konnectors/linkedin.coffee

```diff
@@ -0,0 +1,328 @@
+
+# Vendor
+request = require 'request'
+https = require 'https'
+url = require 'url'
+async = require 'async'
+cheerio = require 'cheerio'
+log = require('printit')
+    prefix: "Linkedin"
+    date: true
+
+ACCOUNT_TYPE = 'com.linkedin'
+
+# GLOBAL
+agent = 'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:41.0) Gecko/20100101 Firefox/41.0'
+
+# Module
+fetcher = require '../lib/fetcher'
+Contact = require '../models/contact'
+Tag = require '../models/tag'
+ContactHelper = require '../lib/contact_helper'
+CompareContacts = require '../lib/compare_contacts'
+
+
+module.exports =
+
+    name: "Linkedin"
+    slug: "linkedin"
+    description: "konnector description linkedin"
+
+    fields:
+        login: "text"
+        password: "password"
+    models:
+        contact: Contact
+
+    fetch: (requiredFields, callback) ->
+        log.info 'Import started'
+        fetcher.new()
+            .use(retrieveTokens)
+            .use(logIn)
+            .use(retrieveListContact)
+            .use(prepareCozyContacts)
+            .use(retrieveContacts)
+            .args(requiredFields, {}, {})
+            .fetch (err, fields, entries) ->
+                return callback err if err
+                log.info "Import finished"
+                callback()
+
+
+retrieveTokens = (requiredFields, entries, data, next) ->
+    log.info 'Retrieve Tokens'
+    opts =
+        url: 'https://linkedin.com'
+        jar: true
+        headers:
+            'User-Agent': agent
+
+    request.get opts, (err, res, body) ->
+        return next err if err
+
+        if body.status? and body.status == 'error'
+            next new Error(body.status_details)
```

Comment:
If an error occured, nothing should happened. So what follows should be displayed in a else block.